Parse/BuildParseTree.pas

```diff
@@ -2913,6 +2913,9 @@ procedure TBuildParseTree.RecogniseStatement;
   CheckLabelPrefix;
 
   lc := fcTokenList.FirstSolidToken;
+  if lc=nil then
```

Comment:
`lc=nil` this do not follow the style guide concerning operators surrounded by spaces (here the link http://edn.embarcadero.com/article/10280, but surprisingly I have not found this statement there)
So,  I just state: please, surround operators by spaces: `if lc = nil ...` I believe it is more readable.

Reply:
ok